game_clean/copy/start_server.bat

```diff
@@ -1,2 +0,0 @@
-@echo off
-srcds.bat +sv_pure 1 +maxplayers 32 +sv_lan 1 %* < nul

```

Comment:
start_server is separate, to isolate "required" launch flags with optional ones.